README.md

```diff
@@ -128,17 +128,32 @@ Again, creating a correct SSL/TLS certificate authority or generally doing certi
 
 2. Compile logstash-forwarder
 
+Note: Do not use gccgo for this project. If you don't know what that means,
+you're probably OK to ignore this.
+
         git clone git://github.com/elasticsearch/logstash-forwarder.git
         cd logstash-forwarder
         go build
 
+gccgo note: Using gccgo is not recommended because it produces a binary with a
+runtime dependency on libgo. With normal the normal go compiler, this
```

Comment:
Normally, with normal the normal go, we normally normalize the normals, normally. Welcome to normaltown.